modules/ossm-cr-profiles.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-reference-smcp.adoc
+
+[id="ossm-cr-profiles_{context}"]
+= profiles parameters
+
+You can create reusable configurations with `ServiceMeshControlPlane` object profiles. If you do not configure the `profile` setting, {ProductName} uses the default profile.
+
+Here is an example that illustrates the `spec.profiles` parameter for the `ServiceMeshControlPlane` object.
+
+.Example profiles parameters
+[source,yaml]
+----
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: basic
+spec:
+  profiles: YourProfileName
+
```

Comment:
```suggestion
apiVersion: maistra.io/v2
kind: ServiceMeshControlPlane
metadata:
  name: basic
spec:
  profiles: YourProfileName
...
```

Reply:
What is this change?  I've looked it over several times and I can't see what change you're asking for here,